user-data-store-server/pom.xml

```diff
@@ -37,6 +37,7 @@
     </properties>
 
     <build>
+        <finalName>user-data-store</finalName>
```

Comment:
If `server.servlet.context-path` is set, the artifact name could stay `user-data-store-server`. But I am ok with this.

Reply:
I think we should keep the artifact name as before.